contracts/modules/capital/MCR.sol

```diff
@@ -90,6 +90,21 @@ contract MCR is IMCR, MasterAwareV2 {
     previousMCR = IMCR(address(0));
   }
 
+  /**
+   * @dev We need to move the mcr way below the current value otherwise swaps won't work
+   * @dev Remove this code after the tokenomics upgrade.
+   */
+  function teleportMCR() external {
+
+    require(address(previousMCR) == address(0), "MCR: not yet initialized");
```

Comment:
do you need to call this function before we actually upgrade the contract?

Reply:
After the actual upgrade. Alternatively I could add this code to the initialize function but I wanted to make it easier to remove the code afterwards, so this was cleaner from that perspective. If we forget to - we have the deadline check.

Reply:
Will have to make the deadline immutable since it fails the tests because it introduces a time dependency.